app/javascript/components/SignIn.js

```diff
@@ -87,6 +87,7 @@ class SignIn extends Component {
             onChange={ changeHandler('password') }
             onBlur={ validateHandler('password') }
             fullWidth
+            className='signUpEmailInputField'
```

Comment:
Isn't signInEmailInputField a better name ?

Reply:
It is. There were class names on other label components that have signUpEmailInputField as the class name so I just gave the components the same class name to target it in the css without writing the same css in different files. But looking over it, there was just one component so I just changed it all to signInEmailInputField.